rllib/algorithms/ppo/torch/ppo_torch_policy_rl_module.py

```diff
@@ -0,0 +1,283 @@
+import logging
+from typing import Dict, List, Type, Union
+
+import ray
+from ray.rllib.algorithms.ppo.ppo_tf_policy import validate_config
+from ray.rllib.evaluation.postprocessing import (
+    Postprocessing,
+    compute_gae_for_sample_batch,
+)
+from ray.rllib.models.action_dist import ActionDistribution
+from ray.rllib.models.modelv2 import ModelV2
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.policy.torch_mixins import (
+    EntropyCoeffSchedule,
+    KLCoeffMixin,
+    LearningRateSchedule,
+)
+from ray.rllib.policy.policy import Policy
+from ray.rllib.policy.torch_policy_v2 import TorchPolicyV2
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.numpy import convert_to_numpy
+from ray.rllib.utils.torch_utils import (
+    apply_grad_clipping,
+    explained_variance,
+    sequence_mask,
+    warn_if_infinite_kl_divergence,
+)
+from ray.rllib.utils.typing import TensorType
+from ray.rllib.algorithms.ppo.torch.ppo_torch_rl_module import (
+    SimplePPOModule,
+    PPOModuleConfig,
+    FCConfig,
+)
+
+torch, nn = try_import_torch()
+
+logger = logging.getLogger(__name__)
+
+
+class PPOTorchPolicyWithRLModule(
+    LearningRateSchedule,
+    EntropyCoeffSchedule,
+    KLCoeffMixin,
+    TorchPolicyV2,
+):
+    """PyTorch policy class used with PPO.
+
+    This class is copied from PPOTorchPolicy (V1) and is modified to support RLModules.
+    Some subtle differences:
+    - if make_rl_module is implemented by the policy the policy is assumed to be v2 and
+        self.model would be an RLModule
+    - Tower stats no longer belongs to the RLModule
```

Comment:
Can you add here, where this goes instead?

Reply:
yep. added.